PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/BubbledGraphTest.java

```diff
@@ -0,0 +1,21 @@
+package nl.tudelft.pl2016gr2.model;
+
+public class BubbledGraphTest extends GraphTest {
+  
+  /**
+   * Setup, creates a new graph.
+   */
```

Comment:
Can't test this on our project since apparently all inheritances (that I could find) lack javadoc, but disregard my comment then